### ISSUE TITLE
Per domain web folder

### ISSUE THREAD

**Issue by sumair-sprinxle** [Ylianst/MeshCentral#1060]
It would be really helpful to have the possibility to create different UI for different sub domains. This will assist to give different clients different interface

**Comment by Ylianst** [comment]
Good request. This could be useful.

**Comment by Ylianst** [comment]
Just published MeshCentral v0.5.0-t with this feature added in. You can now create a folder `meshcentral-web-[domain]` that contains the "views" and "public" folders and it will be used for that domain. As an other option, you can this to the domain section of config.json:

```
"webviewspath": "c:\mypath\views",
"webpublicpath": "c:\mypath\public"
```

This will also set that views and public files are used for that domain. Let me know if that works.
